### PR TITLE
Lightweight readmes for simorgh

### LINKED ISSUE #2873: Add lightweight readmes to the base of all major directories
**Is your feature request related to a problem? Please describe.**
In https://github.com/bbc/simorgh/pull/2870 we removed the components and containers readmes. They were verbose and inconsistent in that they were the only ones which existed.

This leaves us with no readmes at the bases of important directories

**Describe the solution you'd like**
Supplement the primary documentation with a sentence of two in a readme at the base of every major directory. Emphasis should be on brevity and maintainability of these files.

**Describe alternatives you've considered**
Leave with no readmes, which is adequate but perhaps not ideal - it would be better than out of date documentation though

**Testing notes**
No testing needed this is just markdown files

**Additional context**
We should probably refer to how we work between psammead and simorgh more.

### PR DESCRIPTION
**Pull request by jamesbhobbs**
Resolves #2873

**Overall change:** Clarifies the documentation approach in the repo

**Code changes:**

- Describes colocation of documentation approach
- Adds lightweight documentation to various directories and also adds many links between existing docs

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/components/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Components
+
+In general this should not be used. Components should be written, edited and released via [our component library Psammead](https://github.com/bbc/psammead).
```
Comment (amywalkerdev) [inline src/app/components/README.md]: Can we give an example of where this should be used? Or are we trying to stop people putting components here at all?

Comment (jamesbhobbs) [inline src/app/components/README.md]:
```suggestion
In general this should not be used. Components should be written, edited and released via [our component library Psammead](https://github.com/bbc/psammead). For legacy reasons there are still some components here. One reason we might always leave some here, is if our component/s were so tightly coupled with Simorgh specific logic that factoring it out of the repo would have no value.
```

**Review by amywalkerdev — APPROVED** [review]
All good 👍

File: src/app/components/README.md (hunk 1 above)
Comment (jamesbhobbs) [reply src/app/components/README.md]: Done

File: docs/Code-Standards.md [hunk 2]
```diff
@@ -19,6 +21,8 @@ Please familiarise yourself with our:
 
 - Smaller PRs are better, and it's much better for visibility if you open your PRs as draft PRs as early as possible.
 
+- Documentation should be colocated with code wherever possible, and should always be updated. This means having inline comments, or README.md files at the base of the directory where we believe they are helpful. The priority is light-touch documentation in order to keep our documentation easy to keep up-to-date.
```
Comment (chris-hinds) [inline docs/Code-Standards.md]:
```suggestion
- Documentation should be colocated with code wherever possible, and should always be updated. This means having inline comments, or README.md files at the base of the directory where we believe they are helpful. The priority is light-touch documentation, this helps ensure that our documentation is easy to keep up-to-date.
```

File: src/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# src directory
+
+Everything that will be deployed in production should be in this or the [/public directory](../public/). The latter being the location for the static files.
+
+This does not mean everything in this directory is deployed in production. For example the [Storybook](https://storybook.js.org/) stories and tests are stripped out, but kept here for better Developer eXperience (DX) as colocated files are easier to understand and maintain.
```
Comment (chris-hinds) [inline src/README.md:5]:
```suggestion
This does not mean everything in this directory is deployed in production. For example the [Storybook](https://storybook.js.org/) stories and tests are stripped out, but kept here for better Developer experience (DX) as colocated files are easier to understand and maintain.
```

**Review by chris-hinds — CHANGES_REQUESTED** [review]

File: src/README.md (hunk 3 above)
Comment (jamesbhobbs) [reply src/README.md:5]: That's not a mistake, that's a convention

Comment (chris-hinds) [reply src/README.md:5]: TIL

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**